docs/using-lagoon-the-basics/lagoon-yml.md

```diff
@@ -98,14 +98,13 @@ Drupal & Drush 9: Sync database & files from master environment:
     - run:
         name: Sync DB and Files from master if we are not on master
         command: |
-          if [[ "$LAGOON_GIT_BRANCH" != "master" ]]; then
-            # Only if we don't have a database yet
-            if tables=$(drush sqlq 'show tables;') && [ -z "$tables" ]; then
-                drush sql-sync @lagoon.master @self
-                drush rsync @lagoon.master:%files @self:%files -- --omit-dir-times --no-perms --no-group --no-owner --chmod=ugo=rwX
-            fi
+          # Only if we don't have a database yet
+          if tables=$(drush sqlq 'show tables;') && [ -z "$tables" ]; then
+              drush sql-sync @lagoon.master @self
+              drush rsync @lagoon.master:%files @self:%files -- --omit-dir-times --no-perms --no-group --no-owner --chmod=ugo=rwX
           fi
         service: cli
+        when: LAGOON_GIT_BRANCH != "master"
```

Comment:
Can we replace this with an environmentType check != `production`? Might be safer to have in the docs.

Reply:
yup - i'd agree with that